torch_xla/core/dynamo_bridge.py

```diff
@@ -28,6 +28,25 @@
 ptxla_debug = int(os.environ.get('PT_XLA_DEBUG', '0')) == 1
 
 
+class AliasWithBufferDonorContext(object):
+
+  def __init__(self, should_alias: bool):
```

Comment:
Can you make `should_alias` default to `True`? I think that's going to be the case most of the time when you actually invoke this context manager.